deployment/integration.sh

```diff
@@ -21,10 +21,10 @@ export AWS_PROFILE=$TEAM_ACCOUNT_PROFILE
 
 green='\033[0;32m'
 clear='\033[0m'
-cognitoUserpoolId=`aws cognito-idp list-user-pools --max-results 10 | jq -r '.UserPools[] | select(.Name | contains("team06dbb7fc")) | .Id'`
+cognitoUserpoolId=`aws cognito-idp list-user-pools --max-results 10 --output json | jq -r '.UserPools[] | select(.Name | contains("team06dbb7fc")) | .Id'`
 cognitouserpoolhostedUIdomain=`aws cognito-idp describe-user-pool --user-pool-id $cognitoUserpoolId  |jq -r '.UserPool.Domain'`
```

Comment:
cognitouserpoolhostedUIdomain=`aws cognito-idp describe-user-pool --user-pool-id $cognitoUserpoolId --output json | jq -r '.UserPool.Domain'`

Reply:
Ah totally missed that one due to the fact that the space was missing: `|jq` I used `| jq` as a search option 😅, pr is updated.